package.json

```diff
@@ -2,25 +2,27 @@
   "name": "web-ext",
   "version": "6.8.0",
   "description": "A command line tool to help build, run, and test web extensions",
-  "main": "dist/web-ext.js",
+  "type": "module",
+  "main": "index.js",
   "exports": {
-    "import": "./index.mjs",
-    "require": "./dist/web-ext.js"
+    ".": "./index.js",
+    "./util/adb": "./lib/util/adb.js",
+    "./util/logger": "./lib/util/logger.js"
   },
   "files": [
     "CODE_OF_CONDUCT.md",
     "README.md",
-    "dist/*.js",
+    "lib/**",
     "src/**",
```

Comment:
Do we still need the sources in the npm packages?

Reply:
at the moment the only thing that the src/ modules provides that the lib/ one don't is the flow typing information, I think we may actually remove them, but I was considering looking if it is easy enough to properly export the flowtype data along with removing the src/ files from the package (and so to defer that to a separate PR), does is sound ok to you too?

Reply:
sounds good yeah, also TIL that `README.md` is added by default so we could remove it from this list of `files` (and maybe we don't need `CODE_OF_CONDUCT.md` in the final package.